Sources/XCHammer/XCBuildSettings.swift

```diff
@@ -177,6 +177,7 @@ enum XCSettingCodingKey: String, CodingKey {
     // Hammer Rules
     case codeSignEntitlementsFile = "HAMMER_ENTITLEMENTS_FILE"
     case mobileProvisionProfileFile = "HAMMER_PROFILE_FILE"
+    case frameworkvfsoverlay = "RULES_IOS_VFS_OVERLAY"
```

Comment:
k so I assume we're going to beed to set this on the `rules_ios` side right?

Reply:
It's something that is currently set internally within XCHammer right now: a way to load the VFS at build time